### ISSUE TITLE
Unstable loss, extremely low map

### ISSUE THREAD

**Issue by chenaifang** [balancap/SSD-Tensorflow#361]
**The training parameters are set as follows：**

#!/bin/bash
DATASET_DIR=../tfrecords/train
TRAIN_DIR=../train_model/
CHECKPOINT_PATH=../checkpoints/vgg_16.ckpt
python ../train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=train \
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --checkpoint_model_scope=vgg_16 \
    --checkpoint_exclude_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --trainable_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --save_summaries_secs=60 \
    --save_interval_secs=60 \
    --weight_decay=0.0005 \
    --optimizer=adam \
    --learning_rate=0.01 \
    --learning_rate_decay_factor=0.97 \
    --batch_size=16 \
    --max_number_of_steps=50000 \
    --gpu_memory_fraction=0.9

**loss:**
![2](https://user-images.githubusercontent.com/34065949/62941171-01292400-be08-11e9-887f-c68cc267e490.png)

**test result:**
![1](https://user-images.githubusercontent.com/34065949/62941061-b9a29800-be07-11e9-9eb6-e8cc679b1014.png)

**Comment by chenaifang** [comment]
@balancap Could youplease help me with this problem？

**Comment by beibeiZ** [comment]
how you deal with this problem?

**Comment by navy63** [comment]
The　ｓａｍｅ　ｐｒｏｂｌｅｍ，

**Comment by chenjjjjuu** [comment]
the same

**Comment by DeepWu** [comment]
hello,do you have solve this problem? can you help me?
